device/accton/x86_64-accton_wedge100bf_32qs-r0/montara/switch-sai.conf

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I think this file is used by P4_14 stack only. So, should be Ok to remove at this point.

Reply:
Hi akokhan,

OK, We will remove this file. Thanks